.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Fixes _URL of issue_.
+
+_The first line of your PR should begin with the magic word "Fixes", followed by the full URL of the issue it is addressing. Every pull request must be linked to an issue on github.com/backdrop/backdrop-issues, and most of the rationale, discussion, and explanation should take place in the issue, rather than in the PR._
+
+_Once you have submitted your PR, wait a little bit for the automated tests to run and then check that they have all passed. If not, you should revise your PR until they do._
+
+_Once all of the automated checks have passed, you can mark its issue with the labels "status - has pull request", "PR - needs testing", and "PR - needs code review." This will signify to other Backdrop contributors that it's time to take a look. You can also post comments or queries in the issue comments._
```

Comment:
```suggestion
<!-- Once all of the automated checks have passed, you can mark its issue with the labels "status - has pull request", "PR - needs testing", and "PR - needs code review." This will signify to other Backdrop contributors that it's time to take a look. You can also post comments or queries in the issue comments. -->
```